app/controllers/sessions_controller.rb

```diff
@@ -50,6 +51,10 @@ def create_login_record user
       session[:login_id] = login.id
     end
 
+    def create_failure_record user, user_name
```

Comment:
```suggestion
    def create_fail_record user, user_name
```
Solo para que coincida con el nombre del modelo.

Reply:
mal, me quedo del anterior jajaja